libavcodec/vvc/vvc_cabac.c

```diff
@@ -1935,22 +1988,26 @@ static const uint8_t qstate_translate_table[][2] = {
     { 0, 2 }, { 2, 0 }, { 1, 3 }, { 3, 1 }
 };
 
-static int abs_level_decode(VVCLocalContext *lc, const ResidualCoding *rc, const int xc, const int yc)
-{
-    const int c_rice_param = abs_get_rice_param(rc, xc, yc, 0);
-    int abs_dec_level =  abs_decode(lc, c_rice_param);
+static int dec_abs_level_decode(VVCLocalContext *lc, const ResidualCoding *rc, const int xc, const int yc, int *abs_level) {
```

Comment:
we do not need change the function so much, just changer dec_abs_level to int * and move it to last function paramters